ckanext/spatial/harvested_metadata_fgdc.py

```diff
@@ -41,7 +41,7 @@ def read_value(self, name):
     def get_xml_tree(self):
         if self.xml_tree is None:
             parser = etree.XMLParser(remove_blank_text=True)
-            xml_str = six.ensure_str(self.xml_str)
+            xml_str = self.xml_str.decode(encoding='utf-8')
```

Comment:
This should be changed to `xml_str = str(self.xml_str)` as xml_str is likely a string and decode will fail here.